transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/SelectTransferMethodFragment.java

```diff
@@ -293,6 +304,13 @@ public void showCurrencySelectionDialog(@NonNull final TreeMap<String, String> c
     @Override
     public void showAddTransferMethod(@NonNull final String country, @NonNull final String currency,
             @NonNull final String transferMethodType, @NonNull final String profileType) {
+        Map<String, String> params = new HashMap<>();
+        params.put(InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_COUNTRY, country);
+        params.put(InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_CURRENCY, currency);
+        params.put(InsightEventTag.InsightEventTagEventParams.TRANSFER_METHOD_TYPE, transferMethodType);
```

Comment:
we can make this code cleaner if we do like this collection initialization
```
Map<String, String> params = new HashMap<String, String>() {{
           put("key", "value"); 
}};
```

Reply:
please refer previous comments

Reply:
good catch 🌮 I agree let's change ⚙️ 
Any suggestion to make this cleaner? So the goal is just call insight once in the fragment, like one liner code in each fragment, I also did not like to add constants in fragment